civet/asset_precompiler.py

```diff
@@ -478,6 +481,12 @@ def collect_coffee_and_sass_files():
     # just repeat it here verbatim
     ignore_patterns = ['CVS', '.*', '*~']
 
+    # Optionally exclude more patterns from Civet's watchful eyes.  Examples
+    # include directories for NPM managed packages (node_modules) or Bower
+    # managed packages (bower_components).
+    if additional_ignore_patterns:
+      ignore_patterns += additional_ignore_patterns
```

Comment:
Inconsistent indentation.  Also, I think you can just do `ignore_patterns.extend(additional_ignore_patterns)`